src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -163,7 +163,7 @@ private void EnsureHashExists(FileData file)
             var hasSha256Hash = file.Hashes.Any(x => x.Algorithm == AlgorithmKind.Sha256);
             if (!hasSha256Hash)
             {
-                string hashString = GenerateHash(file.Contents.Text);
+                string hashString = GenerateHash(file.Contents.Binary);
```

Comment:
The spec says that for a text file, you can populate either or both of `fileContent.text` and `fileContent.binary`, and for a binary file, you can only populate `fileContent.binary`. So for a text file, `fileContent.binary` might not be present.

Reply:
Fixed